sites/fast-website/src/app/components/content-placement-container/content-placement-container.template.ts

```diff
@@ -43,26 +38,27 @@ export const ContentPlacementContainerTemplate = html<ContentPlacementContainer>
                         >
                     </site-feature-card>`
                 )}
-            `
-        )}
-        ${when(
-            x => x.section === "community",
-            html<ContentPlacementContainer>`
-                ${repeat(
-                    x => x.communityContentPlacementData,
-                    html`<site-content-placement icon>
-                        <div slot="icon" :innerHTML=${x => x.icon}></div>
-                        <h3>${x => x.header}</h3>
-                        <p slot="body">${x => x.body}</p>
-                        <fast-anchor
-                            slot="action"
-                            appearance="lightweight"
-                            href=${x => x.actionLink}
-                            >${x => x.actionText}</fast-anchor
-                        >
-                    </site-content-placement>`
-                )}
-            `
-        )}
-    </div>
+                <site-card-section> </site-card-section
```

Comment:
extra `<site-card-section>`

Reply:
I wonder how I managed that.  Thanks John!  Cleaned up.